### PR TITLE
update: add tooltip for disabled retry and force retry buttons

### PR DESCRIPTION
**Pull request by Oluwadaminiola**

### CODE REVIEW

**Comment by horlah** [comment]
@Oluwadaminiola You might want to consider using the browser's default tooltip. Add a title to a button element; hover on the button, and the title value should appear as a tooltip.

**Comment by Oluwadaminiola** [comment]
> 

@horlah I've done this

File: web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.ts [hunk 1]
```diff
@@ -265,17 +265,20 @@ export class EventLogsComponent implements OnInit {
...
+			this.isRetrying = false;
 			return;
 		} catch (error) {
+			this.isRetrying = true;
 			return error;
 		}
 	}
 
-	async batchRetryEvent() {
+	async batchReplayEvent() {
```
Comment (horlah) [inline web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.ts:281]: Is there a reason you changed it to replay?
The action being performed is retry. It is the event in event log that is replayed

File: web/ui/dashboard/src/app/private/pages/project/events/event-deliveries/event-deliveries.component.html [hunk 2]
```diff
@@ -217,6 +217,7 @@ <h4 class="mb-10px font-semibold text-12">Deliveries missing?</h4>
 									size="xs"
 									texture="light"
 									[disabled]="event.status !== 'Success' && event.status !== 'Failure' && event.status !== 'Discarded'"
+									[title]="event.status !== 'Failure' && event.status !== 'Success' && event.status !== 'Discarded' ? 'You can not retry an event when it is scheduled or still processing.' : ''"
```
Comment (horlah) [inline web/ui/dashboard/src/app/private/pages/project/events/event-deliveries/event-deliveries.component.html:220]: This should be replay, not retry

**Review by horlah — CHANGES_REQUESTED** [review]

File: web/ui/dashboard/src/app/private/pages/project/events/event-deliveries/event-deliveries.component.html (hunk 2 above)
Comment (Oluwadaminiola) [reply web/ui/dashboard/src/app/private/pages/project/events/event-deliveries/event-deliveries.component.html:220]: This is for an event delivery, aren't event deliveries retried while events are replayed

**Review by horlah — APPROVED** [review]

**Pull request merged**